generators/utils.js

```diff
@@ -595,7 +595,7 @@ function getAllJhipsterConfig(generator, force, basePath = '') {
     const filePath = path.join(basePath || '', '.yo-rc.json');
     if ((force || !configuration.baseName) && jhiCore.FileUtils.doesFileExist(filePath)) {
         let yoRc;
-        if (generator) {
+        if (generator && generator.fs) {
```

Comment:
Stupid question: in which cases is there no `fs` attribute? My understanding of a generator (in JHipster) is that there's always an `fs` attribute because it inherits the BaseGenerator (or GeneratorBase, can't remember which).

Reply:
Maybe only test cases, anyway I added the verification:
https://github.com/jhipster/generator-jhipster/runs/617852159

Reply:
By the way fs is from yeoman-generator.
It's memory based, so any change made to the memory fs will be ignored by getAllJHipsterConfig.
It will be written to disc only at conflict step (after writing and before end).

Reply:
Okay, thanks!